TAC/Meetings/2022/2022-10-20/CCC-TAC-Minutes-2022-10-20.md

```diff
@@ -0,0 +1,138 @@
+# Confidential Computing Consortium
+
+## Minutes of the Technical Advisory Council
+
+*The TAC meets for two hours on alternating Thursdays. All members of the community are welcome to attend and participate.*
+
+### Meeting details
+
+**Date**: Thursday, October 20, 2022
+
+**Time**: 7a-9a US Pacific Time
+
+* **Zoom**: [https://zoom.us/j/184384055](https://zoom.us/j/184384055) (pw: 43690)
+* **Calendar**: [https://calendar.confidentialcomputing.io](https://calendar.confidentialcomputing.io) ([ical](https://calendar.google.com/calendar/ical/c_c0pcihr7n2n1k3a38i32d9ag10%40group.calendar.google.com/public/basic.ics), [Google Calendar](https://calendar.google.com/calendar/u/0/r?cid=c_c0pcihr7n2n1k3a38i32d9ag10@group.calendar.google.com))
+* **Recordings**: [YouTube TAC Playlist](https://www.youtube.com/playlist?list=PLmfkUJc39uMjaB_I1dYW72I44kr9QzG_B)
+
+## Links
+
+* **Code of Conduct**: [code-of-conduct.confidentialcomputing.io](https://code-of-conduct.confidentialcomputing.io)
+* **CCC Charter**: [charter.confidentialcomputing.io](https://charter.confidentialcomputing.io)
+* **LF Training course on DEI**: [Inclusive Open Source Community Orientation (LFC102) (free)](https://training.linuxfoundation.org/training/inclusive-open-source-community-orientation-lfc102/)
+* **Declared project dependencies**: [Google Sheets](https://docs.google.com/spreadsheets/d/1UKnbbGWXYLjnPZsox3zmYo59nv3XSXjePfas5E2fER0/edit#gid=0)
+* **CCC YouTube**: [youtube.confidentialcomputing.io](https://youtube.confidentialcomputing.io)
+* **LFX**: [lfx.linuxfoundation.org](https://lfx.linuxfoundation.org)
+* **Join the CCC**: [join.confidentialcomputing.io](https://join.confidentialcomputing.io)
+* **Contact the CCC**: [confidentialcomputing.io/contact-us/](https://confidentialcomputing.io/contact-us/)
+
+## Agenda and Minutes
+
+### Welcome
+   * Dave Thaler (DT) opened the call at 7:03am PT
+   * DT welcomed the members of the TAC and reviewed the values of the CCC and the antitrust policy of the Linux Foundation
+   * Kurt Taylor (KT) captured the meeting minutes
+   * DT reviewed the agenda
+
+### Roll Call / Attendance
+
+***Voting members of the TAC***
+
+*Per the [charter](https://charter.confidentialcomputing.io), all [CCC Premier members](https://confidentialcomputing.io/members/) receive one vote on the TAC. Quorum for votes is at least 50% of voting members present.*
+
+   * Accenture - Giuseppe Giordano
+   * Ant Group - Hongliang Tian (Tate)
+   * Arm - Thomas Fossati + / Michael Lu
+   * Google - Cfir Cohen / Catalin Sandu +
+   * Huawei - Zhipeng (Howard) Huang +
+   * Intel - Dan Middleton + / Simon Johnson
+   * Meta - Eric Northup / Shankaran Gnanashanmugam
+   * Microsoft - Dave Thaler (TAC chair) +
+   * Red Hat - Lily Sturmann + / Dimitrios Pendarakis
+
+   " + " indicates attending
+
+***Other Attendees***
+
+   * Ranjit Narjala (Google)
+   * Alec Fernandez (Microsoft)
+   * Steve Van Lare
+   * Richard Searle
+   * Johannes Kreisler
```

Comment:
Can we add the affiliations for these attendees?

Reply:
Added Steve and Richard, Johannes did not introduce, cannot resolve affiliation